doc/modules/mod_pubsub.md

```diff
@@ -94,15 +94,15 @@ This permits creating a Key-Value list to define a custom node plugin on a given
 E.g. pair `{"urn:xmpp:microblog:0", "mb"}` will use module `node_mb` instead of `node_pep` when the specified namespace is used.
 
 
-#### `modules.mod_pubsub.default_node_config`
+### `modules.mod_pubsub.default_node_config`
 * **Syntax:** TOML table with the following values: string, boolean or non-negative integer.
 * **Default:** `[]`
 * **Example:** `default_node_config = {deliver_payloads = true, max_payload_size, 10000, node_type = "leaf"}`
 
 Overrides the default node configuration, regradless of the node plugin.
 Node configuration still uses the default configuration defined by the node plugin, and overrides any items by the value defined in this configurable list.
 
-#### `modules.mod_pubsub.item_publisher`
+### `modules.mod_pubsub.item_publisher`
```

Comment:
`modules.mod_pubsub.sync_broadcast`  in line 113 (the option below that was not modified so I'm commenting here) has one indentation level to much